.github/workflows/regression-report.yml

```diff
@@ -81,30 +99,177 @@ jobs:
         env:
           COMMENT_BODY: ${{ github.event.comment.body }}
         run: |
-          CONFIG_BUNDLE_PATH=$(echo "$COMMENT_BODY" | \
+          mkdir -p .ci-temp
```

Comment:
Why this required at this point ?

If some shell misses it, please add inside shell script

Reply:
removed